stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -11,7 +11,8 @@
 //===----------------------------------------------------------------------===//
 
 %import gyb
-
+@available(*, unavailable, renamed: "UnsafeMutableRawBufferPointer")
+typealias UnsafeRawMutableBufferPointer = UnsafeMutableRawBufferPointer
```

Comment:
These are the lines we need. It's preferable not to make whitespace changes in the rest of the file alongside a real change such as this.

Reply:
Okay, I will restore the whitespace changes in the file to their original state.